### PR TITLE
Mint 1981 Fix linter warnings

### PR DESCRIPTION
**Pull request by olegdizus**
Using deepscan.io for check code quality. Fixed 57 problems. Some of them could affect UI or generate errors. Also this PR improves readability and bring some minor optimisations.

### CODE REVIEW

File: src/layouts/partials/NotificationContent/NotificationContent.jsx [hunk 1]
```diff
@@ -188,8 +188,8 @@ class NotificationContent extends PureComponent {
               {...notice.message()}
             />
           </div>
-          {details && details.map((item, index) => (
-            <div key={index} styleName='infoRow'>
+          {details && details.map((item) => (
+            <div key={item.label} styleName='infoRow'>
```
Comment (wAISw) [inline src/layouts/partials/NotificationContent/NotificationContent.jsx]: If labels will be the same?

File: src/layouts/partials/NotificationContent/NotificationContent.jsx [hunk 2]
```diff
@@ -153,9 +153,9 @@ class NotificationContent extends PureComponent {
             <div styleName='infoLabel'><Translate value={`${prefix}.hash`} /></div>
             {hash && <div styleName='infoValue'>{hash}</div>}
           </div>
-          {details && details.map((item, index) => {
+          {details && details.map((item) => {
             return (
-              <div key={index} styleName='infoRow'>
+              <div key={item.label} styleName='infoRow'>
```
Comment (wAISw) [inline src/layouts/partials/NotificationContent/NotificationContent.jsx]: If labels will be the same?

File: packages/core/dao/PollInterfaceDAO.js [hunk 3]
```diff
@@ -49,7 +49,7 @@ export default class PollInterfaceDAO extends AbstractContractDAO {
     options.map((option, i) => {
       const value = new BigNumber(values[i])
       if (!value.isZero()) {
-        votes = votes.set(option.toString(), new Amount(value || 0, 'TIME'))
+        votes = votes.set(option.toString(), new Amount(value.isZero() ? 0 : value, 'TIME'))
```
[the remark this answers is not in the document]
Comment (olegdizus) [reply packages/core/dao/PollInterfaceDAO.js]: In this case should be `new Amount(value, 'TIME'). Agree?

File: packages/core/dao/VotingManagerDAO.js [hunk 4]
```diff
@@ -166,7 +166,7 @@ export default class VotingManagerDAO extends AbstractContractDAO {
               memberOption,
             })
 
-            const pollFiles = poll && await ipfs.get(poll.files)
+            const pollFiles = poll.files && await ipfs.get(poll.files)
```
[the remark this answers is not in the document]
Comment (olegdizus) [reply packages/core/dao/VotingManagerDAO.js]: `poll` is instance of `PollModel` that constructed before. So `poll` always `true` at this point, but `poll.files` may be null.

File: packages/core/dao/WavesDAO.js [hunk 5]
```diff
@@ -192,7 +192,7 @@ export default class WavesDAO extends EventEmitter {
       this.emit(EVENT_UPDATE_BALANCE, {
         account,
         time,
-        balance: readBalanceValue(this._symbol, balance, this._asset),
+        balance: readBalanceValue(this._symbol, balance),
```
[the remark this answers is not in the document]
Comment (olegdizus) [reply packages/core/dao/WavesDAO.js]: Mike commented below in file:
```
//TODO WHY WE NEED SYMBOL AND ASSET DESCRIPTION IF SYMBOL IS ENOUGH
function readBalanceValue (symbol, balance /*asset = null*/) {
  // @todo assets
  // console.log('readBalanceValue: ', symbol, balance, asset)
  // if (asset) {
  //   return balance.assets[asset]
  // }
  return balance.balance
}
```

File: packages/core/redux/persistAccount/utils.js [hunk 6]
```diff
@@ -20,12 +20,12 @@ export const getAddress = (address, /*hexFormat = false*/) => {
   return address//`${ hexFormat ? '0x' : ''}${address}`
 }
 
-export const getAccountAddress = (account: AccountEntryModel, hexFormat = false) => {
-  return account && account.encrypted && account.encrypted[0] && getAddress(account.encrypted[0].address, hexFormat) || ''
+export const getAccountAddress = (account: AccountEntryModel) => {
```
[the remark this answers is not in the document]
Comment (olegdizus) [reply packages/core/redux/persistAccount/utils.js]: Because it is unused, sure.

**Pull request merged**